CMakeLists.txt

```diff
@@ -239,15 +239,15 @@ endif()
 find_package(Doxygen)
 option(ENABLE_DOCS "Enable building documentation." ON)
 if(DOXYGEN_FOUND AND ENABLE_DOCS)
-    set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES "${PROJECT_BINARY_DIR}/dev-docs/_build")
+    set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES "${CMAKE_CURRENT_BINARY_DIR}/dev-docs/_build")
```

Comment:
It looks like this change would help if H3 were included as a subproject, is that right?

Reply:
Yes. Just always a good idea to use `CMAKE_CURRENT_SOURCE_DIR`/`CMAKE_CURRENT_BINARY_DIR` as opposed to their `PROJECT_...` counterparts.